src/HealthChecks.AzureServiceBus/Configuration/AzureServiceBusQueueMessagesCountThresholdHealthCheckOptions.cs

```diff
@@ -0,0 +1,29 @@
+namespace HealthChecks.AzureServiceBus.Configuration;
+
+/// <summary>
+/// Configuration options for <see cref="AzureServiceBusQueueMessageCountThresholdHealthCheck"/>.
+/// </summary>
+public class AzureServiceBusQueueMessagesCountThresholdHealthCheckOptions : AzureServiceBusQueueHealthCheckOptions
+{
+    /// <summary>
+    /// Indicates if dead letter messages queue or active messages queue should be checked.
+    /// </summary>
+    public bool CheckDeadLetterMessages { get; }

```

Comment:
Why not both?

Reply:
I mean why do you need that setting at all?

Reply:
I suggest to check both types of messages (if corresponding threshold was specified) in a single health check.